### ISSUE TITLE
av1.md and wavpack.md not included in codecs RFC

### ISSUE THREAD

**Issue by mbunkus** [ietf-wg-cellar/matroska-specification#391]
The contents of the two files `codec/av1.md` and `wavpack.md` aren't included in the codecs RFC at all. For WavPack there's at least a short section in `codec_specs.md` that only links to the separate HTML file. Run the following command if you want to make sure I'm not totally bonkers:

```
grep -Eri 'codec/|wavpack' | grep -vE '^(codec/av1.md|wavpack.md):'
```

If there are no good reasons to keep their content out of the RFC, I suggest we either move the content both files into `codec_specs.md` or somehow include both file s in the build process so that the resulting codecs RFC will still have the IDs sorted alphabetically.

**Comment by robUx4** [comment]
av1.md is a standalone document. It currently doesn't build to a RFC but maybe at some point it could.

wavpack.md could be integrated in the bigger codecs document.

**Comment by robUx4** [comment]
AV1 was merged in #857 with most of the juicy part from av1.md (and corrections).

wavpack still needs to be more defined in `codec_specs.md`.

**Comment by robUx4** [comment]
Wavpack was merged in #920
